Expensify/App
# Do not show an empty space above the chats in the LHN

@marcaaron, @shawnborton, @tgolen can you review?  

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/146508

### Tests
1. Open Native chat. Have A few pinned chats in the LHN.
2. Notices there's no big space between the search bar and the pinned chats:
<img width="302" alt="Screen Shot 2020-11-24 at 11 40 10 AM" src="https://user-images.githubusercontent.com/19366280/100083820-3f562580-2e4a-11eb-9d50-1df191e26827.png">
3. Search for a user in the chat switcher. Make sure the first result does not overlap with the search bar:
<img width="299" alt="Screen Shot 2020-11-24 at 11 40 02 AM" src="https://user-images.githubusercontent.com/19366280/100083863-5268f580-2e4a-11eb-97fc-9de51c1b4ebb.png">
4. Check this works correctly across platforms.
